pandas/io/formats/style_render.py

```diff
@@ -698,7 +700,7 @@ def format(
         Using a ``formatter`` with LaTeX ``escape``.
 
         >>> df = pd.DataFrame([["123"], ["~ ^"], ["$%#"]])
-        >>> s = df.style.format("\\textbf{{{}}}", escape="latex").to_latex()
+        >>> s = df.style.format("\\textbf{{{}}}", escape="latex").to_latex() # doctest: +SKIP
```

Comment:
`to_latex()` actually produces a string result.. maybe this could be doctested.

Reply:
The command line output I got by running this test is not pretty printed actually as it is in the docs: 
```
________ [doctest] pandas.io.formats.style_render.StylerRenderer.format ________
694         ...
695         <td .. ><a href="a.com/&lt;div&gt;&lt;/div&gt;">&lt;div&gt;&lt;/div&gt;</a></td>
696         <td .. ><a href="a.com/&#34;A&amp;B&#34;">&#34;A&amp;B&#34;</a></td>
697         <td .. >NA</td>
698         ...
699 
700         Using a ``formatter`` with LaTeX ``escape``.
701 
702         >>> df = pd.DataFrame([["123"], ["~ ^"], ["$%#"]])
703         >>> df.style.format("\\textbf{{{}}}", escape="latex").to_latex()
Expected:
    \begin{tabular}{ll}
    {} & {0} \\
    0 & \textbf{123} \\
    1 & \textbf{\textasciitilde \space \textasciicircum } \\
    2 & \textbf{\$\%\#} \\
    \end{tabular}
Got:
    '\\begin{tabular}{ll}\n{} & {0} \\\\\n0 & \\textbf{123} \\\\\n1 & \\textbf{\\textasciitilde \\space \\textasciicircum } \\\\\n2 & \\textbf{\\$\\%\\#} \\\\\n\\end{tabular}\n'

```
Please let me know if I am missing something here, or have done something incorrectly. 
Thanks.